src/main/webapp/projects_research.html

```diff
@@ -60,6 +60,10 @@ <h2> Google Maps API </h2>
     <p> This is where I will experimented with a an API.
         enjoy :D </p>
     <div id="map"></div>
+    <p> Servlet button from walk through. </p>
+    <button onclick = "getServerMessages()">Servlet Demo </button>
+    <div id="quote-container"></div>
```

Comment:
Do you mean to make this div a `<ul>` or `<ol>` element?

Right now, createListElement appends `<li>` elements inside the `<div>`, which isn't really valid HTML (although most browsers will understand what you're doing and display it properly).

Reply:
corrected